src/lib/viewers/image/ImageViewer.js

```diff
@@ -339,8 +344,8 @@ class ImageViewer extends ImageBaseViewer {
         this.imageEl.style.top = `${topPadding}px`;
 
         // Fix the scroll position of the image to be centered
-        this.wrapperEl.scrollLeft = (this.wrapperEl.scrollWidth - viewport.width) / 2;
-        this.wrapperEl.scrollTop = (this.wrapperEl.scrollHeight - viewport.height) / 2;
+        this.wrapperEl.scrollLeft = (this.imageEl.clientWidth - viewport.width) / 2;
```

Comment:
There is more inside the wrapper now (annotations), so we need to look at the size of the actual image for our scroll reference.